bundles/org.openhab.binding.broadlink/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="broadlink"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<channel-type id="command">
+		<item-type>String</item-type>
+		<label>Command</label>
+		<description>The IR/RF command sent to the device</description>
+		<category>Text</category>
+	</channel-type>
+	<channel-type id="temperature">
+		<item-type>Number</item-type>
```

Comment:
```suggestion
		<item-type>Number:Temperature</item-type>
```
You will need to delete the thing and re-add it after making this change, also any items create will need to be recreated. So best this is changed to UOM so a breaking change is not introduced later on.